pyproject.toml

```diff
@@ -1,19 +1,20 @@
 [tool.poetry]
 name = "virga"
-version = "1.4.0"
+version = "1.5.0"
 description = "Dynamic IPA sidecar app generation and plugins."
 authors = ["Elias Gabriel <elias.gabriel@indicodata.ai>"]
 readme = "virga/_cli/templates/boilerplate/README.md"
 include = ["CHANGELOG.md"]
 
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
```

Comment:
with the way you use typing... should this be ^3.10?

Reply:
for example https://github.com/IndicoDataSolutions/virga/pull/26/files#diff-9b07cacc50e7439647b6e1b1399d226e8b582891f7c314a94723cc2a80a109b6R43 will break in 3.8

Reply:
it should be 3.8, any other typing incompats are typos or bugs

eagle eyes - though the fact thats a typo is unfortunate, because it requires i add another typing import at the top of file which will screw up all the patch files 😢

Reply:
:( guess to validate in the tests, you could always use a py3:8 dockerfile and maybe drone would catch incompatibilities too?

Reply:
oh just kidding you do hmmm

Reply:
the tests only check the plugins and generations, they dont check the validity of the generated projects / some calls are mocked for speed (aside from helm, which checks that the generated chart renders)

i suppose a way, might be overkill, to do that would be to get rid the gen mocks, include some unit tests in the boilerplate project, and after generation then invoke the venv to invoke another pytest on those sub-unit tests

Reply:
makes my head hurt

Reply:
to prevent this from becoming monolithic, im making the unit/integration test pondering/spec/refactor a different ticket